vendor/vendor.json

```diff
@@ -149,7 +149,7 @@
 			"revisionTime": "2017-03-30T08:48:43Z"
 		},
 		{
-			"checksumSHA1": "DCjamhJdcLN963jm6/yFoDbJvT4=",
```

Comment:
Sorry about that I actually ran the tests locally which made govender sync run and then I must have committed it while amending my changes.

One question since running tests changes vendor.json should we run the tests locally or add vendor.json to `.gitignore`